packages/react-app/src/modules/airdrop/AirdropModule.tsx

```diff
@@ -0,0 +1,398 @@
+  const [txInProgress, setTxInProgress] = React.useState(false);
+  const [distributorContract, setDistributorContract] = React.useState(null);
+  const [user, setUser] = React.useState(null);
+
+  const [showAnswers, setShowAnswers] = React.useState([]);
+
+  const toggleShowAnswer = (id) => {
+    let newShowAnswers = [...showAnswers];
+    newShowAnswers[id] = !newShowAnswers[id];
+
+    setShowAnswers(newShowAnswers);
+  }
+
+  React.useEffect(() => {
+    if (!onCoreNetwork) {
+      const url = window.location.origin;
+      window.location.replace(`${url}/#/notavailable`);
+    }
+  })
+
+  React.useEffect(() => {
+    if (!!(library && account)) {
+      let signer = library.getSigner(account);
+      console.log(abis.distributor)
+      const signerInstance = new ethers.Contract(addresses.distributor, abis.distributor, signer);
+      setDistributorContract(signerInstance);
+      // const NFTRewardsInstance = new ethers.Contract(addresses.NFTRewards, abis.NFTRewards, signer);
+      // setNFTRewardsContract(NFTRewardsInstance);
+    }
+  }, [account, library]);
+
+  React.useEffect(() => {
+    if (distributorContract) {
+      checkClaim();
+    }
+  }, [account, distributorContract]);
+
+  // to check wh
+  const checkClaim = async () => {
+    let user = await AirdropHelper.verifyAddress(account, distributorContract);
+    setUser(user)
+    if (user)
+      setLoading(false)
+  }
+
+  // to claim
+  const handleClaim = async (user) => {
+    if (distributorContract) {
+      setTxInProgress(true)
+      let sendWithTxPromise
+      sendWithTxPromise = await distributorContract.claim(user.index, user.account, user.amount, user.proof)
+      const tx = await sendWithTxPromise;
+      console.log(tx);
+      console.log("waiting for tx to finish");
+      let txToast = toast.dark(<LoaderToast msg="Waiting for Confirmation..." color="#35c5f3" />, {
+        position: "bottom-right",
+        autoClose: false,
+        hideProgressBar: true,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+      });
+      try {
+        await library.waitForTransaction(tx.hash);
+
+        toast.update(txToast, {
+          render: "Transaction Completed!",
+          type: toast.TYPE.SUCCESS,
+          autoClose: 5000
+        });
+
+        setTxInProgress(false);
+      }
+      catch (e) {
+        toast.update(txToast, {
+          render: "Transaction Failed! (" + e.name + ")",
+          type: toast.TYPE.ERROR,
+          autoClose: 5000
+        });
+
+        setTxInProgress(false);
+      }
+      setLoading(false);
+    }
+  }
+
+  // toast customize
+  const LoaderToast = ({ msg, color }) => (
+    <Toaster>
+      <Loader
+        type="Oval"
+        color={color}
+        height={30}
+        width={30}
+      />
+      <ToasterMsg>{msg}</ToasterMsg>
+    </Toaster>
+  )
+
+  const [darkMode, setDarkMode] = useState(false);
+
+
+  return (
+    <Container>
```

Comment:
this container can be removed as we already have a Container in the AirdropPage. the css can also be shifted there.
This can be done in gov module chat module too.